### ISSUE TITLE
Failed to compile styleguide on 3.0.0

### ISSUE THREAD

**Issue by Sumolari** [viljamis/vue-design-system#87]
Just tried to run version 3.0.0: I cloned the repo, checked out 3.0.0 tag and run `yarn install` and `yarn start`.

The main page (`localhost:8080`)  works fine, however the styleguide (`localhost:6060`) can't be built:

```
FAIL  Failed to compile

./src/elements/SvgIcon.vue?vue&type=script&lang=js)
Module not found: Can't resolve '@/assets/icons/' in 'PATH_TO_CLONE/src/elements'
./docs/components/status/Components.vue?vue&type=script&lang=js)
Module not found: Can't resolve '@/assets/tokens/tokens.raw.json' in 'PATH_TO_CLONE/docs/components/status'
./docs/components/tokens/Color.vue?vue&type=script&lang=js)
Module not found: Can't resolve '@/assets/tokens/tokens.raw.json' in 'PATH_TO_CLONE/docs/components/tokens'
./docs/components/tokens/FontSize.vue?vue&type=script&lang=js)
Module not found: Can't resolve '@/assets/tokens/tokens.raw.json' in 'PATH_TO_CLONE/docs/components/tokens'
./docs/components/tokens/Spacing.vue?vue&type=script&lang=js)
Module not found: Can't resolve '@/assets/tokens/tokens.raw.json' in 'PATH_TO_CLONE/docs/components/tokens'
./docs/components/tokens/All.vue?vue&type=script&lang=js)
Module not found: Can't resolve '@/assets/tokens/tokens.raw.json' in 'PATH_TO_CLONE/docs/components/tokens'
./docs/components/status/Components.vue?vue&type=script&lang=js)
Module not found: Can't resolve '@/elements/' in 'PATH_TO_CLONE/docs/components/status'
./docs/components/status/Components.vue?vue&type=script&lang=js)
Module not found: Can't resolve '@/patterns/' in 'PATH_TO_CLONE/docs/components/status'
./docs/components/status/Components.vue?vue&type=script&lang=js)
Module not found: Can't resolve '@/templates/' in 'PATH_TO_CLONE/docs/components/status'
```

Checked out tag `2.1.4` and it worked fine.

I'm running Node 8.9.1 on macOS 10.13.4 with Yarn 1.7.0.

**Comment by viljamis** [comment]
@Sumolari Please follow the instructions in [Getting Started](https://github.com/viljamis/vue-design-system/wiki/getting-started). You need to use `npm install` & `npm start` for things to work.

**Comment by Sumolari** [comment]
Looks like `yarn install` is the culprit

- If dependencies are installed using `npm install` server can be started both with `yarn start` or `npm start`
- If dependencies are installed using `yarn install` then neither `yarn start` nor `npm start` work

Given that version 2.1.4 worked with `yarn` there must be some issue introduced lately either in this package or in one of the dependencies that changed from 2.1.4 to 3.0.0.

**Comment by viljamis** [comment]
@Sumolari @phaust This should be fixed with 3.0.1 release. It was caused by one of the NPM dependencies of the project.
